code/game/gamemodes/changeling/powers/biodegrade.dm

```diff
@@ -57,7 +57,17 @@
 		to_chat(user, "<span class='warning'>We secrete acidic enzymes from our skin and begin melting our cocoon...</span>")
 		addtimer(CALLBACK(src, .proc/dissolve_cocoon, user, C), 2.5 SECONDS) //Very short because it's just webs
 		used = TRUE
-
+	for(var/obj/item/grab/G in user.grabbed_by)
+		var/mob/living/carbon/M = G.assailant
+		user.visible_message("<span class='warning'>[user] spits acid at [M]'s face and slips out of their grab!</span>")
+		M.Stun(1) //Drops the grab
+		M.apply_damage(5, BURN, "head", M.run_armor_check("head", "melee"))
+		user.SetStunned(0) //This only triggers if they are grabbed, to have them break out of the grab, without the large stun time. If you use biodegrade as an antistun without being grabbed, it will not work
+		user.SetWeakened(0)
+		user.lying = FALSE
+		user.update_canmove()
```

Comment:
`update_canmove()` is already called by `SetWeakened`

Reply:
Fixed